### PR TITLE
Resframe 2997 ensure dev cert

### PR DESCRIPTION
**Pull request by JoeARO**

### CODE REVIEW

File: rel/runtime_config.exs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+use Mix.Config
+
+cert = System.gen_env("DEV_CERT_PEM")
```
Comment (james-bowers) [inline rel/runtime_config.exs]: did you mean?
```suggestion
cert = System.get_env("DEV_CERT_PEM")
```

Comment (JoeARO) [reply rel/runtime_config.exs]: Yes I did... for the second commit I extracted it to the variable and must have mistyped... should copy paste it next time 🤦‍♂

Comment (JoeARO) [reply rel/runtime_config.exs]: I was hoping you hadn't spotted that before me but you were too damn quick :P

File: rel/runtime_config.exs [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+if cert == nil do
+  raise "Your developer certificate is not set correctly please ensure you set it to the environment variable $DEV_CERT_PEM"
+end
+
+if File.exists?(cert) == false do
+  raise "The DEV_CERT_PEM environment variable is set, however the file path is incorrect"
+end
+
+config :mozart_fetcher,
+        Keyword.new([{String.to_atom(String.downcase("DEV_CERT_PEM")), System.get_env("DEV_CERT_PEM")}])
```
Comment (james-bowers) [inline rel/runtime_config.exs]: does this not work then? 🤔 
```suggestion
        Keyword.new([:dev_cert_pem, System.get_env("DEV_CERT_PEM")}])
```

Comment (JoeARO) [reply rel/runtime_config.exs]: Sorry copy pasted that bit from belfrage one

File: rel/runtime_config.exs [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+if cert == nil do
+  raise "Your developer certificate is not set correctly please ensure you set it to the environment variable $DEV_CERT_PEM"
+end
+
+if File.exists?(cert) == false do
+  raise "The DEV_CERT_PEM environment variable is set, however the file path is incorrect"
+end
+
+config :mozart_fetcher,
+        Keyword.new([:dev_cert_pem, System.get_env("DEV_CERT_PEM")}])
```
Comment (james-bowers) [inline rel/runtime_config.exs]: not a big deal, just thought of this:
```suggestion
        Keyword.new([:dev_cert_pem, cert}])
```

**Comment by samfrench** [comment]
Closing due to being open a while.

**Pull request closed without merging**